seq2science/rules/quantification.smk

```diff
@@ -268,70 +210,180 @@ elif config["quantifier"] == "kallistobus":
               opts="-p -t "$(wc -l {input.r1} | grep -Po '^\d+' | awk '{{print int($1/4)}}')
             fi
             fastq_pair $opts {output.intermediates1} >> {log} 2>&1
+            gzip -c {output.intermediates3[0]} > {output.reads[0]} 2> {log}
+            gzip -c {output.intermediates3[1]} > {output.reads[1]} 2>> {log}
```

Comment:
I'm not sure if we should add gzip to the env of this rule.... There don't really seem to be any real conda ones.. Is it like kinda always installed?

Reply:
I could not find it in any official Conda channels either. I did a quick screen of the pkg manifests of some Ubuntu versions and bgzip it is not part of the default install.